.github/workflows/pr-realm-js.yml

```diff
@@ -386,47 +366,57 @@ jobs:
           if-no-files-found: error
 
   merge-prebuilds:
+    name: Merge prebuild artifacts
     runs-on: ubuntu-latest
     needs:  [prebuild-node, prebuild-apple, prebuild-android]
     steps:
-      - name: Merge prebuild artifacts into a single artifact
-        uses: actions/upload-artifact/merge@v4
+      - uses: actions/upload-artifact/merge@v4
         with:
           name: prebuilds
           pattern: '*-prebuild'
 
-  integration-tests:
-    name: Test ${{ matrix.variant.environment }} on ${{ matrix.variant.os }} (${{matrix.variant.target}})
-    needs: [generate-jsi, build-ts, prebuild-node, prebuild-apple, prebuild-android]
-    if: ${{ success() || failure() }}
-    env:
-      REALM_DISABLE_ANALYTICS: 1
-      MOCHA_REMOTE_TIMEOUT: 60000
-      LONG_TIMEOUT: 300000 # 5 minutes
-      MOCHA_REMOTE_REPORTER: mocha-github-actions-reporter
-      MOCHA_REMOTE_EXIT_ON_ERROR: true
-      SPAWN_LOGCAT: true
-      BAAS_BRANCH: master
-      # Pin the Xcode version
-      DEVELOPER_DIR: /Applications/Xcode_14.3.1.app
-      IOS_DEVICE_NAME: iPhone 14
-    runs-on: ${{ matrix.variant.runner }}
+  node-electron-tests:
+    name: Test ${{ matrix.environment }} on ${{ matrix.runner }} (${{matrix.script_name}})
+    needs:
+      - generate-jsi
```

Comment:
Why is JSI needed for node? Shouldn't it be `node-prebuild`?

Reply:
Great catch - `generate-jsi` is not needed for the Node / Electron tests 👍